### PR TITLE
updating the domain from 'spinoperator.dev' to 'spinkube.dev'

### PR DESCRIPTION
**Pull request by asteurer**
Per @calebschoepp, this still needs a SKIP, and possibly some peripheral design work. Submitting the PR to start a those conversations

### CODE REVIEW

File: charts/spin-operator/templates/mutating-webhook-configuration.yaml [hunk 1]
```diff
@@ -13,12 +13,13 @@ webhooks:
     service:
       name: '{{ include "spin-operator.fullname" . }}-webhook-service'
       namespace: '{{ .Release.Namespace }}'
+      # TODO @asteurer: Should this filepath be updated?
```
Comment (calebschoepp) [inline charts/spin-operator/templates/mutating-webhook-configuration.yaml:16]: Yes, the `spinoperator-dev` is the domain of the resource with `.` replaced by `-` to be a valid url path.

File: internal/webhook/spinapp_defaulting.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // nolint:lll
-//+kubebuilder:webhook:path=/mutate-core-spinoperator-dev-v1alpha1-spinapp,mutating=true,failurePolicy=fail,sideEffects=None,groups=core.spinoperator.dev,resources=spinapps,verbs=create;update,versions=v1alpha1,name=mspinapp.kb.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/mutate-core-spinoperator-dev-v1alpha1-spinapp,mutating=true,failurePolicy=fail,sideEffects=None,groups=core.spinkube.dev,resources=spinapps,verbs=create;update,versions=v1alpha1,name=mspinapp.kb.io,admissionReviewVersions=v1
```
Comment (calebschoepp) [inline internal/webhook/spinapp_defaulting.go:14]: It needs to be changed here too.

File: internal/webhook/spinapp_validating.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // nolint:lll
-//+kubebuilder:webhook:path=/validate-core-spinoperator-dev-v1alpha1-spinapp,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.spinoperator.dev,resources=spinapps,verbs=create;update,versions=v1alpha1,name=vspinapp.kb.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/validate-core-spinoperator-dev-v1alpha1-spinapp,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.spinkube.dev,resources=spinapps,verbs=create;update,versions=v1alpha1,name=vspinapp.kb.io,admissionReviewVersions=v1
```
Comment (calebschoepp) [inline internal/webhook/spinapp_validating.go:18]: And here

File: internal/webhook/spinappexecutor_defaulting.go [hunk 4]
```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // nolint:lll
-//+kubebuilder:webhook:path=/mutate-core-spinoperator-dev-v1alpha1-spinappexecutor,mutating=true,failurePolicy=fail,sideEffects=None,groups=core.spinoperator.dev,resources=spinappexecutors,verbs=create;update,versions=v1alpha1,name=mspinappexecutor.kb.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/mutate-core-spinoperator-dev-v1alpha1-spinappexecutor,mutating=true,failurePolicy=fail,sideEffects=None,groups=core.spinkube.dev,resources=spinappexecutors,verbs=create;update,versions=v1alpha1,name=mspinappexecutor.kb.io,admissionReviewVersions=v1
```
Comment (calebschoepp) [inline internal/webhook/spinappexecutor_defaulting.go:13]: And here

File: internal/webhook/spinappexecutor_validating.go [hunk 5]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 // nolint:lll
-//+kubebuilder:webhook:path=/validate-core-spinoperator-dev-v1alpha1-spinappexecutor,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.spinoperator.dev,resources=spinappexecutors,verbs=create;update,versions=v1alpha1,name=vspinappexecutor.kb.io,admissionReviewVersions=v1
+//+kubebuilder:webhook:path=/validate-core-spinoperator-dev-v1alpha1-spinappexecutor,mutating=false,failurePolicy=fail,sideEffects=None,groups=core.spinkube.dev,resources=spinappexecutors,verbs=create;update,versions=v1alpha1,name=vspinappexecutor.kb.io,admissionReviewVersions=v1
```
Comment (calebschoepp) [inline internal/webhook/spinappexecutor_validating.go:15]: And here

**Review by calebschoepp — COMMENTED** [review]
Good work, thank you

**Comment by adamreese** [comment]
@asteurer (or maintainer) Can you move this PR from draft to ready for review?

**Pull request closed without merging**